examples/showcase/painter/components/brush.js

```diff
@@ -68,6 +73,16 @@ AFRAME.registerComponent('brush', {
     this.painting = false;
   },
 
+  onButtonChanged: function (evt) {
+    if (!this.data.enabled) { return; }
+    if (!this.painting) { return; }
+    if (evt.detail.state.value === 1) {
+      this.data.size = this.originalSize;
```

Comment:
don't modify the `this.data.size` It will be overridden if the component property changes. Have a `this.sizeFactor` variable that is applied below
```this.stroke.setSize(this.data.size * this.sizeFactor)```

Reply:
Nice catch. SizeFactor added, take a look when you get a chance. And thanks for the review!